website/events-listener/src/gear/connection.ts

```diff
@@ -24,7 +27,13 @@ export async function connectToGearNode() {
   eventListenerLogger.info(`Connected to ${chain} with genesis ${genesis}`);
 
   const unsub = await listen(api, genesis, ({ key, value }) => {
-    kafkaProducer.send(key, value, genesis);
+    const sendByKafkaTopicInput: SendByKafkaTopicInput = {
+      topic: KAFKA_TOPICS.EVENTS,
```

Comment:
don't you think that it will be better to specify topic in `listener` then here you will send just data from the callback?

Reply:
In this case you can send data for `update.message.payload` topic here